imagefiles/dockcross

```diff
@@ -193,13 +193,24 @@ else
     HOST_PWD=$PWD
 fi
 
+# Mount Additional Volumes
+if [ -z "$SSH_DIR" ]; then
+    SSH_DIR="$HOME/.ssh"
+fi
+
+HOST_VOLUMES=
+if [ -e "$SSH_DIR" ]; then
+    HOST_VOLUMES+="-v $SSH_DIR:/home/$(id -un)/.ssh"
```

Comment:
Will this work when using docker on windows and macosx ? Or should it be as a special case ?